NEWS.md

```diff
@@ -47,6 +47,9 @@ Standard library changes
   supports precompilation and setting compatibility.
 #### LinearAlgebra
 
+- The `Eigen` type now supports more methods, such as matrix functions
+  (which return `Eigen` objects), adjoints and matrix-vector product.
+  `inv(::Eigen)` now produces an `Eigen` type instead of a matrix.
```

Comment:
Is this enough for this to be considered a "minor change"?

FWIW, I did an audit of my personal code, and while I never assume that `inv(::Eigen)` returns a matrix, I _have_ relied on the fact that `inv(::LU)` returns a matrix.

In any case, it might be nice to say explicitly how one can continue to get the inverse matrix from `::Eigen`, if that's what a user wants.

Reply:
Good point. x-ref #32137. And certainly this requires a pkgeval run if we go with that change.

Reply:
As far as I understood the rules were more relaxed about breaking changes for stdlibs? I can either special-case `inv` to return a matrix (and we can change it for 2.0) or make the change now with a more explicit NEWS. Can somebody run pkgeval, just to get an idea of how many people rely on this? (I'd be surprised, but...)

Reply:
@antoine-levitt You can run pkgeval yourself! It's as easy as:

@nanosoldier `runtests()`

See https://github.com/JuliaCI/Nanosoldier.jl#readme for more info